packages/kbn-scout/src/servers/run_kibana_server.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import Path from 'path';
+import Os from 'os';
+import { v4 as uuidv4 } from 'uuid';
+import type { ProcRunner } from '@kbn/dev-proc-runner';
+import { REPO_ROOT } from '@kbn/repo-info';
+import { parseRawFlags, getArgValue, remapPluginPaths, DedicatedTaskRunner } from '@kbn/test';
+import { Config } from '../config';
+
+export async function runKibanaServer(options: {
+  procs: ProcRunner;
+  config: Config;
+  installDir?: string;
+  extraKbnOpts?: string[];
+  logsDir?: string;
+  onEarlyExit?: (msg: string) => void;
+  inspect?: boolean;
+}) {
+  const { config, procs } = options;
+  const runOptions = options.config.get('kbnTestServer.runOptions');
+  const installDir = runOptions.alwaysUseSource ? undefined : options.installDir;
+  const devMode = !installDir;
+  const useTaskRunner = options.config.get('kbnTestServer.useDedicatedTaskRunner');
+
+  const procRunnerOpts = {
+    cwd: installDir || REPO_ROOT,
+    cmd: installDir
+      ? process.platform.startsWith('win')
+        ? Path.resolve(installDir, 'bin/kibana.bat')
+        : Path.resolve(installDir, 'bin/kibana')
+      : process.execPath,
+    env: {
+      FORCE_COLOR: 1,
+      ...process.env,
+      ...options.config.get('kbnTestServer.env'),
+    },
+    wait: runOptions.wait,
+    onEarlyExit: options.onEarlyExit,
+  };
+
+  const prefixArgs = devMode
+    ? [Path.relative(procRunnerOpts.cwd, Path.resolve(REPO_ROOT, 'scripts/kibana'))]
+    : [];
+
+  if (options.inspect) {
+    prefixArgs.unshift('--inspect');
```

Comment:
Have you tried to debug a test with the node debugger to make sure the setup works?

Reply:
removing since it is not used